hibernate-rx-core/src/main/java/org/hibernate/rx/engine/loading/impl/RxLoadContexts.java

```diff
@@ -0,0 +1,64 @@
+package org.hibernate.rx.engine.loading.impl;
+
+import org.hibernate.collection.spi.PersistentCollection;
+import org.hibernate.engine.loading.internal.CollectionLoadContext;
+import org.hibernate.engine.loading.internal.EntityLoadContext;
+import org.hibernate.engine.loading.internal.LoadContexts;
+import org.hibernate.engine.spi.CollectionKey;
+import org.hibernate.engine.spi.PersistenceContext;
+import org.hibernate.persister.collection.CollectionPersister;
+
+import java.sql.ResultSet;
+
+public class RxLoadContexts extends LoadContexts {
```

Comment:
@DavideD What is the purpose of this class? It's not documented, and it doesn't appear to do anything interesting.

Reply:
Sorry, it's not needed. Just something I forgot to delete